pushy/src/main/java/com/relayrides/pushy/apns/ApnsPushNotification.java

```diff
@@ -120,4 +120,23 @@
      * @since 0.5
      */
     String getTopic();
+
+    /**
+     * <p>
+     * Returns the id of the notification. This is generally used for notification management
```

Comment:
I think we need to expand/clarify these docs, but I can take responsibility for that.